src/analysisd/analysisd.c

```diff
@@ -531,7 +534,11 @@ int main_analysisd(int argc, char **argv)
                     if (!test_config) {
                         mdebug1("Reading decoder file %s.", *decodersfiles);
                     }
-                    if (!ReadDecodeXML(*decodersfiles)) {
+                    if (!ReadDecodeXML(*decodersfiles, &msg)) {
+                        if (msg) {
+                            printf("%s", msg);
```

Comment:
It should be a `merror` or `mwarn` call instead of a `printf` call

Reply:
I have a problem with this.
the output of the function, saves a set of msg, e.g.
```
*msg = "
analisysd.c:100 at _loadmemory(): ERROR: (1104): Maximum string size reached for: %s. '%s'\n
.. maybe an other warning msg...
analisysd.c:150 at ReadDecodeXML(): ERROR: (1230): Invalid element in the configuration: '%s'\n
"
```
This msg store in debug mode all the information necessary to send over the socket
The following output is valid for the log file?

```
" analisysd.c:50 at ReadDecodeXML(): INFO: call ReadDecodeXML result in the following errors/warning:
analisysd.c:500 at _loadmemory(): ERROR: (1104): Maximum string size reached for: %s. '%s'\n
.. maybe an other warning msg...
analisysd.c:300 at ReadDecodeXML(): ERROR: (1230): Invalid element in the configuration: '%s'\n
"
```

Reply:
Same as above